### PR TITLE
Adding support for Sorani Kurdish 

### PR DESCRIPTION
**Pull request by Hrazhan**
The current RTL implementation does not support Kurdish letters which is a type of Arabic script. Since TRDG uses the arabic-reshaper library, I could easily solve this by passing the "language" parameter to the ArabicReshaper class. 

- Along with the RTL parameter to True `--language` parameter  should be set to "ckb"
- I also added a dictionary of Kurdish words
- Wrote a test function with a font supporting Kurdish letters

### CODE REVIEW

File: trdg/generators/from_strings.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import os
 
+from numpy.lib.function_base import delete
```
Comment (Belval) [inline trdg/generators/from_strings.py:3]: Is this used anywhere?

**Review by Belval — CHANGES_REQUESTED** [review]
Overall I'd be inclined to merge this feature. Please address my comment.

File: trdg/generators/from_strings.py (hunk 1 above)
Comment (Hrazhan) [reply trdg/generators/from_strings.py:3]: Hey, I haven't used that anywhere

**Comment by SilvaQ** [comment]
@Belval 
ug 语言的生成也需要这个PR， 我暂先本地调整了，望早点合并

**Review by Belval — APPROVED** [review]

**Comment by Belval** [comment]
Will test and merge this week, thank you for your contribution!

**Comment by Belval** [comment]
Thank you for your contribution.

**Pull request merged**